frontend/src/components/templates/Modal.tsx

```diff
@@ -40,7 +45,9 @@ export default function Modal({ children, onClickClose }: Props) {
             leaveFrom="opacity-100 translate-y-0 sm:scale-100"
             leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
           >
-            <Dialog.Panel className="modal-panel">
+            <Dialog.Panel
+              className={`modal-panel ${fitContent && 'h-fit w-fit'}`}
```

Comment:
```suggestion
              className={classnames(
                  'modal-panel',
                  fitContent && 'h-fit w-fit'
              )}
```

요런 상황에서 utils에 있는 classNames 함수 쓰시면 됩니다..! 문자열 안에서 `&&`나 `? : ` 사용하면 렌더링 했을 때 class 속성에 `false`나 `undefined`가 찍히더라구요